.github/workflows/matrix.js

```diff
@@ -144,7 +134,7 @@ include.forEach(v => {
     // so it might reveal missing synchronization in TestNG code
     v.name += ', stress JIT';
     jvmArgs.push('-XX:+UnlockDiagnosticVMOptions');
-    if (v.jdk.version >= 8) {
+    if (v.jdk.version >= 11) {
```

Comment:
You can remove the condition because it is always true now.

Reply:
Fixed and pushed the amended commit.